.github/workflows/build_etiss.yml

```diff
@@ -0,0 +1,133 @@
+name: Build ETISS
+
+on:
+  repository_dispatch:
+    types: [s4e-cdl-event, seal5-event]
+#push:
```

Comment:
remove comment

Reply:
Or comment it in, if we want the workflow to be executed when etiss has new commits (which might be a good idea...)

Reply:
commented this in.